src/_data/sidenav.yml

```diff
@@ -142,8 +142,27 @@
           permalink: /docs/development/tools/android-studio
         - title: Visual Studio Code
           permalink: /docs/development/tools/vs-code
-        - title: Dart DevTools
-          permalink: https://flutter.github.io/devtools/
+        - title: DevTools
+          permalink: /docs/development/tools/devtools
+          children:
+            - title: Overview
+              permalink: /docs/development/tools/devtools/overview
+            - title: Install from Android Studio
```

Comment:
same comment as before regarding Android Studio vs Android Studio / IntelliJ

Reply:
Initially, I was a little leery of bloating the side nav, but I went ahead and added it.  :)